src/Tribe/Views/V2/View.php

```diff
@@ -258,8 +258,10 @@ public static function make_for_rest( \WP_REST_Request $request ) {
 			$params['prev_url'] = untrailingslashit( $params['prev_url'] );
 		}
 
-		$slug       = Arr::get( $params, 'view', false );
-		$url_object = Url::from_url_and_params( Arr::get( $params, 'url' ), $params );
+		$slug = Arr::get( $params, 'view', false );
+		// Convert the URL to lowercase to make sure the rewrite rules, all lowercase, will match it.
+		$url  = Arr::get( $params, 'url' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 2 spaces

<sub>hash: 71046e868662a64407e8f8c908cd6729</sub>